docker/docker/src/main/java/org/arquillian/cube/docker/impl/await/LogScanningAwaitStrategy.java

```diff
@@ -72,78 +66,57 @@ public boolean isStdErr() {
     @Override
     public boolean await() {
         final DockerClient client = dockerClientExecutor.getDockerClient();
-        
-        Info info = client.infoCmd().exec();
-        
-        int since = parseTimestamp(info.getSystemTime());
-        
-        final LogContainerResultCallback callback = new LogContainerResultCallback(since);
-        
-        return Ping.ping(pollIterations, getSleepTime(), getTimeUnit(), new PingCommand() {
-            @Override
-            public boolean call() {
-                try {
-                    client.logContainerCmd(cube.getId())
-                            .withStdOut(stdOut).withStdErr(stdErr)
-                            .withTimestamps(true).withSince(callback.getLastTimestamp())
-                            .exec(callback).awaitCompletion();
-                }
-                catch (InterruptedException e) {
-                    // do nothing
-                }
-                
-                return callback.isFound();
+        final CountDownLatch containerUp = new CountDownLatch(1);
+
+        try (final LogContainerResultCallback callback = new LogContainerResultCallback(containerUp)) {
+            try {
+
+                client.logContainerCmd(cube.getId())
+                        .withStdErr(true)
+                        .withStdOut(true)
+                        .withFollowStream(true)
+                        .exec(callback);
+
+                boolean result = containerUp.await(this.timeout, TimeUnit.SECONDS);
+                return result;
```

Comment:
I put in different statements because I wanted to add something between but now I cannot remember what :) I fox for now